From 877bdf95fdfdac840a7a664362ec69b85c59ecb0 Mon Sep 17 00:00:00 2001
From: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Date: Fri, 3 Jul 2020 09:37:51 +0900
Subject: [PATCH 04/32] ctl: improve documentation for identifier of control
 element

In documentation, there're two ways relevant to the identifier of control
element. However, the case of combination has the lack of parameters.

This commit improves documentation in this point.

Fixes: f3c24de8c0df ("ctl: add an overview for design of ALSA control interface")
Signed-off-by: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 src/control/control.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/control/control.c b/src/control/control.c
index 19528ae30a0b..e21e8f1d2621 100644
--- a/src/control/control.c
+++ b/src/control/control.c
@@ -49,8 +49,8 @@ are managed according to below model.
    - An element includes some members to have a value. The value of each member
      can be changed by both of userspace applications and drivers in kernel.
 
-Each element can be identified by two ways; a combination of name and index, or
-numerical number (numid).
+Each element can be identified by two ways; the numerical number (numid), or the
+combination of interface, device, subdevice, name, and index.
 
 The type of element set is one of integer, integerr64, boolean, enumerators,
 bytes and IEC958 structure. This indicates the type of value for each member in
-- 
2.16.4

